app/src/main/java/me/saket/dank/widgets/IndentedLayout.java

```diff
@@ -37,14 +84,11 @@ public IndentedLayout(Context context, @Nullable AttributeSet attrs) {
 
     TypedArray attributes = context.obtainStyledAttributes(attrs, R.styleable.IndentedLayout);
     spacePerDepthPx = attributes.getDimensionPixelSize(R.styleable.IndentedLayout_spacePerDepth, dpToPx(DEFAULT_SPACING_PER_DEPTH_DP, context));
-    int indentationLineColor = attributes.getColor(R.styleable.IndentedLayout_indentationLineColor, Color.LTGRAY);
```

Comment:
Could rename attribute to `defaultIndentationLineColor` and store it for use as a fallback if colors are disabled.